composer.lock

```diff
@@ -22140,7 +22140,6 @@
     "aliases": [],
     "minimum-stability": "dev",
     "stability-flags": {
-        "drupal/better_exposed_filters": 10,
```

Comment:
Hmm. I'm not sure this lock file should have changed either. 🤔

Reply:
I think this one is okay. I noticed my translation branch has the same change after doing some `composer requires`. so this appears to be expected.